recipe/meta.yaml

```diff
@@ -58,6 +110,6 @@ about:
   dev_url: https://github.com/pyca/service-identity
 
 extra:
-  feedstock-name: service_identity
```

Comment:
Please keep this here

Reply:
[8e0e65b](https://github.com/conda-forge/service_identity-feedstock/pull/7/commits/8e0e65bd042bcfe6c46292f477a1d5e0747e7596)